.deepsource.toml

```diff
@@ -0,0 +1,22 @@
+version = 1
```

Comment:
Appreciate your help!

Could you exclude this file in this PR? I think we need further discussion about integrating Deep Source.

Reply:
Sure! I will make the changes and mention you right here. 

If you can provide any feedback on the DeepSource, and any particular reasons for excluding it, that would be great!